auth-password-store.el

```diff
@@ -45,10 +45,18 @@
 See `auth-source-search' for details on SPEC."
   (cl-assert (or (null type) (eq type (oref backend type)))
              t "Invalid password-store search: %s %s")
-  (when (listp host)
+  (when (and host (listp host))
+    (warn "auth-pass-search ignores all but first host in spec.")
```

Comment:
`auth-pass-search` -> `auth-password-store` (the user shouldn't have to care about implementation)